src/vmdeps.txt

```diff
@@ -27,3 +27,5 @@ cryptsetup
 # filesystems/storage
 gdisk xfsprogs e2fsprogs dosfstools btrfs-progs
 
+# needed for basic CA support
+ca-certificates
```

Comment:
Longer term it'd be nicer to entirely avoid doing networking in supermin - basically we pull content from the container and then operate on it via the VM.  But obviously not the case today.